pkg/registry/registry.go

```diff
@@ -107,8 +107,8 @@ func (or OktetoRegistry) IsOktetoRegistry(image string) bool {
 
 func (or OktetoRegistry) IsGlobalRegistry(image string) bool {
 	expandedImage := or.imageCtrl.expandImageRegistries(image)
-	expandedGlobalImage := fmt.Sprintf("%s/%s", or.config.GetRegistryURL(), or.imageCtrl.config.GetGlobalNamespace())
-	return strings.HasPrefix(expandedImage, fmt.Sprintf("%s/", expandedGlobalImage))
```

Comment:
It seems in master was already fixed?

Reply:
it was fixed, but I think it's more reabable by having 1 `fmt.Sprintf`

Reply:
Personally, I agree with Andrea's POV